tests/lib/rules/indent.js

```diff
@@ -289,7 +291,20 @@ eslintTester.addRuleTest("lib/rules/indent", {
                 "        break;\n" +
                 "}",
             options: [4, {indentSwitchCase: true}],
-            errors: expectedErrors([[13, 4], [15, 4], [17, 4]])
+            errors: expectedErrors([[5, 8], [8, 8]])
+        },
+        {
+            code:
+                "switch (a) {\n" +
+                "case '1':\n" +
+                "b();\n" +
+                "break;\n" +
+                "default:\n" +
+                "c();\n" +
+                "break;\n" +
+                "}",
+            options: [4, {indentSwitchCase: true}],
+            errors: expectedErrors([[3, 4], [4, 4], [6, 4], [7, 4]])
```

Comment:
Exactly, that was the comment I was referring to. Final decision seems to be only about `break` that should be indented by default in the same way as other statements in case body, but nothing about `case` relative to `switch` itself.